en-GB.lproj/Main.strings

```diff
@@ -117,37 +117,37 @@
 "MCD-u6-GW4.normalTitle" = "Open Cydia";
 
 /* Class = "UIButton"; normalTitle = "Restart SpringBoard"; ObjectID = "MKZ-Zb-eq1"; */
-"MKZ-Zb-eq1.normalTitle" = "Restart SpringBoard";
+"MKZ-Zb-eq1.normalTitle" = "Herstart SpringBoard";
 
 /* Class = "UIButton"; normalTitle = "View Source Code"; ObjectID = "MpC-W0-1eR"; */
-"MpC-W0-1eR.normalTitle" = "View Source Code";
+"MpC-W0-1eR.normalTitle" = "Laat Source Code zien";
 
 /* Class = "UILabel"; text = "Restore RootFS"; ObjectID = "Mpp-g3-bNI"; */
-"Mpp-g3-bNI.text" = "Restore RootFS";
+"Mpp-g3-bNI.text" = "Herstel RootFS";
 
 /* Class = "UILabel"; text = "Load Tweaks"; ObjectID = "MyV-qW-zdt"; */
-"MyV-qW-zdt.text" = "Load Tweaks";
+"MyV-qW-zdt.text" = "Laad Tweaks";
 
 /* Class = "UIButton"; normalTitle = "@umanghere (Development)"; ObjectID = "N6S-5q-UzE"; */
-"N6S-5q-UzE.normalTitle" = "@umanghere (Development)";
+"N6S-5q-UzE.normalTitle" = "@umanghere (Ontwikkelaar)";
 
 /* Class = "UILabel"; text = "Hide Log Window"; ObjectID = "NGj-U7-Rke"; */
-"NGj-U7-Rke.text" = "Hide Log Window";
+"NGj-U7-Rke.text" = "Verberg Log Window";
 
 /* Class = "UIButton"; normalTitle = "Button"; ObjectID = "NHc-vO-PPf"; */
 "NHc-vO-PPf.normalTitle" = "Button";
 
 /* Class = "UILabel"; text = "Max Memory Limit"; ObjectID = "PFy-VA-xz7"; */
-"PFy-VA-xz7.text" = "Max Memory Limit";
+"PFy-VA-xz7.text" = "Maximaal Memory Limiet";
 
 /* Class = "UILabel"; text = "Set get-task-allow"; ObjectID = "S6S-NE-WJ9"; */
-"S6S-NE-WJ9.text" = "Set get-task-allow";
+"S6S-NE-WJ9.text" = "zet get-task-allow";
```

Comment:
"Stel get-task-allow in" would be a more fitting translation of the English version.